flink-formats/flink-protobuf/src/test/proto/test_simple_noouter_nomulti.proto

```diff
@@ -21,18 +21,17 @@ package org.apache.flink.formats.protobuf.proto;
 option java_package = "org.apache.flink.formats.protobuf.testproto";
 
 message SimpleTestNoouterNomulti {
```

Comment:
What about rename this file to test_simple_nomulti.proto as you do?

Reply:
Also set multiple_files=false explicitly ?

Reply:
https://github.com/apache/flink/pull/21613/commits/5fb41bb51d838e61dcb323822aca45c880b09d27
Changed proto file and test case names and specified multiple_files options explicitly.